frontend/components/heart_graph.jsx

```diff
@@ -3,132 +3,45 @@ var LineChart = require('react-d3-basic').LineChart;
 
 var HeartGraph = React.createClass({
   componentWillReceiveProps: function(){
-    //must have at least 6 recorded elements
-    if(this.props.chartData.length > 5){
-      if(this.isAccuracyAchieved()){
-        this.props.handleAccuracyAchieved();
-      }
+    //must have at least X recorded elements to obtain a 'valid accuracy'
+    //LAST_X_HEART_RATES in tap_recorder.js
```

Comment:
update comment to better communicate that X  in line 6 is LAST_X_HEART_RATES which can be found in tap_recorder.js